### PR TITLE
implemented CIFAR10 Classifier with Loss and Accuracy plots

### PR DESCRIPTION
**Pull request by Rocksen1683**
Here's my bootcamp submission for the computer vision team at UWARG. I implemented a CIFAR-10 Image Classifier with an accuracy of 82.97%. The model is a sequential keras model that was trained for 25 epochs and the Accuracy and Loss plots were plotted using Pyplot. PFA both of the plots and the training screen.

Please reach out to me for any issues and fixes!
image

### CODE REVIEW

**Review by mika-m-shaw — APPROVED** [review]
Hey! Great job on completing the CV bootcamp! Everything looks great, I honestly have no feedback (a tiny thing might be to call 'run_functions()' 'main()' just for conventional purposes). Our first CV meeting is this thurs where we will go through onboarding and future tasks (on discord). Welcome to the team :)

**Comment by ekpraveen03** [comment]
Thanks for the feedback, will definitely join the meeting on Thursday. Excited to contribute to the team :))
